doc/whats_new.rst

```diff
@@ -31,6 +31,14 @@ Changelog
 New features
 ............
 
+   - :class:`model_selection.GridSearchCV`,
+     :class:`model_selection.RandomizedSearchCV` now support multimetric
```

Comment:
simultaneous evaluation of multiple metrics? Again, I don't think we need to invent a word.